### PR TITLE
[FLOC-2365] Update Readme Content

### PR DESCRIPTION
**Pull request by davidoliverSP2**
The wording for the Flocker introduction (lines 9-18) has been taken from the v1 marketing material, which also features here: https://docs.clusterhq.com/en/1.0.0/introduction/index.html#what-is-flocker

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,21 +5,25 @@ Flocker |coveralls|
 
 .. |coveralls| image:: https://coveralls.io/repos/ClusterHQ/flocker/badge.png
   :target: https://coveralls.io/r/ClusterHQ/flocker
-  :alt: 'Buildbot build coverage status'
 
-Flocker is a data volume manager and multi-host Docker cluster management tool.
-With it you can control your data using the same tools you use for your stateless applications by harnessing the power of ZFS on Linux.
-This means that you can run your databases, queues and key-value stores in Docker and move them around as easily as the rest of your app.
+Flocker is an open-source Container Data Volume Manager for your Dockerized applications.
```
Comment (wallrj) [inline README.rst]: "open-source" or "open source"

It'd be nice to be consistent:

```
(master)(update-readmecontent-FLOC-2365 ? =)[~/.../flocker/docs]$ ack 'open-source'
introduction/index.rst
8:Flocker is an open-source Container Data Volume Manager for your Dockerized applications.
(master)(update-readmecontent-FLOC-2365 ? =)[~/.../flocker/docs]$ ack 'open source'
faq/index.rst
52:We are interested in expanding libswarm to include support for filesystems and are talking with the various open source projects about the best way to collaborate on storage and networking for volumes.
```

Though wikipedia isn't https://en.wikipedia.org/wiki/Open_source

**Comment by wallrj** [comment]
Thanks @davidoliverSP2 

This sounds fine and 
* I checked that the modified link now works.
* other link check failures are due to a bug in our link checker....so fine.

Please merge.

```
(master)(update-readmecontent-FLOC-2365 ? =)[~/.../flocker/docs]$ grep README  _build/linkcheck/output.txt 
README.rst:1: [broken] https://docs.clusterhq.com/en/latest/using/tutorial/index.html: <urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
README.rst:26: [broken] http://docs.clusterhq.com/en/latest/gettinginvolved/contributing.html#talk-to-us: <urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
README.rst:35: [broken] https://docs.clusterhq.com/en/latest/using/tutorial/index.html: <urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
README.rst:35: [broken] https://docs.clusterhq.com/en/latest/using/installing/index.html: <urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
README.rst:35: [broken] https://docs.clusterhq.com/en/latest/introduction/index.html: <urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
README.rst:23: [redirected with Found] https://twistedmatrix.com to https://twistedmatrix.com/trac/
README.rst:41: [redirected with Found] https://tox.readthedocs.org/ to https://tox.readthedocs.org/en/latest/

```

**Comment by exarkun** [comment]
@wallrj did you notice if these broken links were being reported on the link checker prior to the fix?

**Comment by wallrj** [comment]
They're not normally reported because README isn't covered by by our link checker
 * https://clusterhq.atlassian.net/browse/FLOC-1352

I symlinked README.rst into the docs directory and ran the link checker manually.

**Comment by exarkun** [comment]
I wonder if we can convince Sphinx to check this for us as part of the normal doc/link check... Can you file an issue for this?

**Comment by wallrj** [comment]
See issue link above.

**Comment by exarkun** [comment]
Pfft. :(  Sorry.

**Pull request merged**